app/src/main/res/values/strings.xml

```diff
@@ -55,6 +55,7 @@
     <string name="connected_count">Connected: %s of %s online</string>
     <string name="list_of_nodes">A list of nodes in the network</string>
     <string name="update_firmware">Update Firmware</string>
+    <string name="connected">Connected to radio (%s)</string>
```

Comment:
This whole pull request is great (and I'll reply to your kind email tomorrow - currently drinking beer and watching a movie ;-) ).  Can you make one change though?  change this resource name back to connected_to, because I don't want to lose the various language translations others have donated in values_XX.

Reply:
My bad, I've forgot to remove (%s) from this new label. no changes for **connected_to**. Did not see it when I test it because the device was not connected.

I will fix it and add missing french translation

Reply:
heh!  last night when I was sleeping I briefly woke and realized oh!  it probably was a new string and the %s was probably just an oops.  No worries thanks.  And OMG a pull request for french would be awesome.